### ISSUE TITLE
CKEditor LaTeX support

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/ocw-studio#1482]
As an OCW course author, I'd like to be able to edit existing LaTeX in courses and create new LaTeX expressions, including mathematical expressions and chemical equations.

#### Designs and Mockups

From https://github.com/isaul32/ckeditor5-math

<img width="1471" alt="Screen Shot 2022-09-15 at 9 08 18 AM" src="https://user-images.githubusercontent.com/9010790/190412002-051fc79f-b1bd-4865-8809-0a3837e16947.png">

#### Acceptance Criteria:

- [ ] Be able to edit and add new math expressions

#### Context

_This issue is the result of discussions following #1469._

**Comment by ChristopherChudzicki** [comment]
**Note:** The latest version of https://github.com/isaul32/ckeditor5-math uses CKEditor5 v34. We probably need to use a slightly older version of that plugin since we are on v31 iirc.

**Comment by ChristopherChudzicki** [comment]
@Wassaf-Shahzad I looked at the mathjax plugin a bit more after your slack question. I modified the one of the examples on their github github: https://jsfiddle.net/a4zesopf/

There you can see the HTML generated by their plugin is:
```html
<p>Hello world.&nbsp;&nbsp;
    <script type="math/tex">E = mc^2</script>…And also
</p>
<script
    type="math/tex; mode=display">e^{x} = 1 + x + \frac{x^2}{2!} + \cdots</script>
<p>And some chemistry:&nbsp;
    <script type="math/tex">\ce{N2 + 3H2 -> 2 NH3}</script>
</p>
```

And the corresponding Markdown that we would want would be 
```
Hello world \(E = mc^2\) and also

\[e^{x} = 1 + x + \frac{x^2}{2!} + \cdots\]

And some chemistry \(\ce{N2 + 3H2 -> 2 NH3}\)
```

So I think we just need a markdown plugin that:
- converts `<script type="math/tex">LaTeX</script>` to `\(LaTeX\)` (w/ round parens)and vice versa
- converts `<script type="math/tex; mode=display">LaTeX</script>` to `\[LaTeX\]` (w/ square brackets) and vice versa

**Comment by ChristopherChudzicki** [comment]
Closed by #1522
